### PR TITLE
remove restriction on `SELECT INTO FILE` in mysql-compatibility.md

### PR DESCRIPTION
**Pull request by Minorli**
### What is changed, added or deleted? (Required)

* `SELECT INTO FILE` 语法  -->根据测试，4.0 版本是支持此功能的。

Manually cherry-picked to https://github.com/pingcap/docs-cn/pull/3792.

### Which TiDB version(s) do your changes apply to? (Required)

- [ ] master (the latest development version)
- [x ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: mysql-compatibility.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ aliases: ['/docs-cn/stable/reference/mysql-compatibility/']
 * `CREATE TEMPORARY TABLE` 语法
 * `CHECK TABLE` 语法
 * `CHECKSUM TABLE` 语法
-* `SELECT INTO FILE` 语法
+* `SELECT INTO FILE` 语法 --> 经测试，此功能在 4.0 版本得到了支持。
```
Comment (TomShawn) [inline mysql-compatibility.md:38]:
```suggestion
```

**Comment by Minorli** [comment]
@tshqin 天爽请帮忙 review 一下啦，谢谢~

**Comment by tshqin** [comment]
LGTM

**Pull request merged**